modules/op-advantages-and-disadvantages-of-non-versioned-and-versioned-cluster-tasks.adoc

```diff
@@ -0,0 +1,30 @@
+// This module is part of the following assembly:
+//
+// *cicd/pipelines/managing-nonversioned-and-versioned-cluster-tasks.adoc
+:_content-type: CONCEPT
+[id="advantages-and-disadvantages-of-non-versioned-and-versioned-cluster-tasks_{context}"]
+= Advantages and disadvantages of non-versioned and versioned cluster tasks
+
+.Advantages and disadvantages of non-versioned and versioned cluster tasks
+[options="header"]
+|===
+
+| Cluster task variant | Advantages | Disadvantages
+
+| Non-versioned cluster task 
+a| 
+* If you prefer {pipelines-shortname} with latest updates and bug fixes, use the non-versioned cluster task variant. 
+* The non-versioned variants are automatically upgraded during Operator upgrades, and  minimize redundant resources.  
+a| {pipelines-shortname} using the non-versioned cluster tasks may break after an Operator upgrade, as the automatically upgraded cluster tasks may be backward incompatible. 
+
+| Versioned cluster task 
+a| 
+* If you prefer stable {pipelines-shortname} in production, use the versioned cluster task variant. 
+* The old version is retained on the cluster even after the new version of a cluster task is installed, which results in wastage of cluster resources.
+a| 
+* If you continue using an old version of a cluster task, you may miss the latest features and critical security updates.
```

Comment:
```suggestion
* If you continue using an old version of a cluster task, you might miss the latest features and critical security updates.
```
Consider "may" a banned word. Consider using "might" or other alternatives instead. Search the IBM Style Guide for more information.